api/src/pcapi/scheduled_tasks/commands.py

```diff
@@ -295,8 +295,8 @@ def delete_suspended_accounts_after_withdrawal_period() -> None:
     if not FeatureToggle.ALLOW_ACCOUNT_REACTIVATION:
         return
 
-    users = fraud_api.get_suspended_upon_user_request_accounts_since(settings.DELETE_SUSPENDED_ACCOUNTS_SINCE)
-    for user in users:
+    query = fraud_api.get_suspended_upon_user_request_accounts_since(settings.DELETE_SUSPENDED_ACCOUNTS_SINCE)
```

Comment:
pourquoi retourner une query plutôt que la liste des utilisateurs ?

Reply:
A l'utilisation (itérer dessus), ça revient au même, on récupère des utilisateurs au final.
Et l'avantage d'une Query c'est qu'on la réutiliser (ajouter un yield_per pour une autre utilisation, ajouter un filtre).

Mais si tu penses que ce n'est pas clair ou que ça ne vaut pas le coup, je peux renvoyer directement une liste d'utilisateurs. Je n'ai pas d'avis tranché là-dessus, juste une légère préférence pour l'option Query.